Documentation/network/kubernetes/ciliumendpointslice.rst

```diff
@@ -104,6 +104,9 @@ Several options are available to adjust the performance and behavior of the CES
 
 * You can also fine-tune rate-limiting settings for the Operator communications with the API-server. Refer to the ``--ces-*`` flags for the ``cilium-operator`` binary.
 
+* You can annotate priority namespaces by setting attribute ``cilium.io/ces-namespace`` to the value “priority”. When dealing with large clusters, the propagation of changes during Network Policy updates can be significantly delayed.
```

Comment:
I'd say `annotation` instead of `attribute`, while it might seem obvious you mean an annotation, it might still be confusing for users less familiar with kubernetes.

Reply:
Thanks, I changed it